plugins/json-logger/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "my-plugin",
+  "version": "0.0.1",
+  "description": "Description of your plugin",
```

Comment:
Fill in these fiels similar to this: https://github.com/amplication/plugins/blob/master/plugins/deployment-helm-chart/package.json

This data is used to upload the plugin to NPM.